dotnet/src/webdriver/PrintOptions.cs

```diff
@@ -199,6 +201,11 @@ internal Dictionary<string, object> ToDictionary()
                 toReturn["shrinkToFit"] = this.shrinkToFit;
             }
 
+            if (this.pageSize == null) 
```

Comment:
Now it is better, which makes unnecessary to keep `this.pageSize` field initialized: https://github.com/SeleniumHQ/selenium/blob/4c2491faa3207cce1e9890bac16e98dfc7e5ee2e/dotnet/src/webdriver/PrintOptions.cs#L55

Reply:
Ah not, it is behavior breaking change. Please keep it as is.